core/src/main/java/org/mapfish/print/map/tiled/AbstractTiledLayer.java

```diff
@@ -90,4 +91,16 @@ public final void prepareRender(
     public final double getImageBufferScaling() {
         return this.imageBufferScaling;
     }
+    
+    @Override
+    public final void cacheResources(final HttpRequestCache httpRequestCache,
+            final MfClientHttpRequestFactory clientHttpRequestFactory, final MapfishMapContext transformer) {
+        final MapfishMapContext layerTransformer = getLayerTransformer(transformer);
+        
+        final double dpi = transformer.getDPI();
+        final TilePreparationTask task = new TilePreparationTask(
+                clientHttpRequestFactory, dpi, layerTransformer,
+                this.tileCacheInformation, httpRequestCache);
+        this.tilePreparationInfo = this.forkJoinPool.invoke(task);
```

Comment:
Does it still make sense to use `forkJoinPool`? The tiles are no longer downloaded at this point, so it would add an overhead to run this in parallel.